### PR TITLE
Result Streaming

### PR DESCRIPTION
**Pull request by anthonydresser**
implements the result streaming front end

### CODE REVIEW

File: extensions/import/package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
     }
   },
   "dependencies": {
-    "dataprotocol-client": "github:Microsoft/sqlops-dataprotocolclient#0.2.7",
+    "dataprotocol-client": "github:Microsoft/sqlops-dataprotocolclient#feat/resultStreaming_release",
```
Comment (kburtram) [inline extensions/import/package.json]: this should probably reference a tag off master prior to merge?

File: src/sql/parts/query/execution/queryRunner.ts [hunk 2]
```diff
@@ -336,7 +348,7 @@ export default class QueryRunner extends Disposable {
 	/**
```
Comment (ranasaria) [inline src/sql/parts/query/execution/queryRunner.ts]: Comment still refers to ResultSetComplete method name.

File: src/sql/parts/query/services/queryEditorService.ts [hunk 3]
```diff
@@ -249,9 +249,9 @@ export class QueryEditorService implements IQueryEditorService {
 				QueryEditorService.editorService.openEditor(newEditorInput, options, group).then((editor) => {
 					resolve(QueryEditorService._onEditorOpened(editor, uri.toString(), undefined, options.pinned));
 				},
-				(error) => {
-					reject(error);
-				});
+					(error) => {
```
Comment (ranasaria) [inline src/sql/parts/query/services/queryEditorService.ts]: Is this whitespace only change useful or inadvertent?

**Comment by kburtram** [comment]
@anthonydresser I think @ranasaria's related SQL Tools Service changes are now published.  Is the front-end side of the change ready to go in today?  If possible, it would be great to pick this up in the `insiders` build we're publishing this afternoon.

**Comment by anthonydresser** [comment]
I'll merge in now; the build failure is from failing vs search tests.

**Pull request merged**